src/redux/books/books.js

```diff
@@ -13,6 +15,27 @@ export const removeBook = id => ({
   id,
 });
 
+export const getBooks = () => async dispatch => {
+  const apiBooks = await fetchApi.get();
+  Object.entries(apiBooks).forEach(([id, [{ category, title }]]) => {
+    dispatch(addBook({ category, title, id }));
+  });
+};
+
+export const postBooks = (payload, method) => async dispatch => {
+  if (method === 'POST') {
+    dispatch(addBook(payload));
+  } else {
+    dispatch(removeBook(payload));
+  }
+  await fetchApi.post(payload, method);
+};
+
+// export const deleteBooks = id => async dispatch => {
+//   dispatch(removeBook(id));
+//   await fetchApi.deleteBook(id);
+// };
+
```

Comment:
- [ ] Let us remove the commented lines in our code to keep our code clean and professional.
![week2](https://user-images.githubusercontent.com/55635977/131584795-fbbc80d3-d175-4dd8-9a9e-2c9f51d36d70.png)